models/product-type/src/transformers.ts

```diff
@@ -0,0 +1,19 @@
+import { Transformer } from '@commercetools-test-data/core';
+import type { TProductType, TProductTypeGraphql } from './types';
+
+const transformers = {
+  default: Transformer<TProductType, TProductType>('default', {
+    buildFields: ['attributes'],
+  }),
+  rest: Transformer<TProductType, TProductType>('rest', {
+    buildFields: ['attributes'],
+  }),
+  graphql: Transformer<TProductType, TProductTypeGraphql>('graphql', {
+    buildFields: ['attributes'],
```

Comment:
This is a fun one. The `attributes` field is `attributeDefinitions` in GraphQL
![image](https://user-images.githubusercontent.com/8136478/218781638-5c164d8f-adc6-43c2-9831-4aa997cab55c.png)

Reply:
🔧  Fixed with [2f4ca81](https://github.com/commercetools/test-data/pull/198/commits/2f4ca8174a0c9595ff0eb70101921a68468f89be)